This crate is unmaintained for many years with license issues and deprecation
warnings, but it is only used for two aarch64 tests.

--- a/src/arch/Cargo.toml
+++ b/src/arch/Cargo.toml
@@ -21,6 +21,3 @@
 logger = { path = "../logger" }
 utils = { path = "../utils" }
 vm-memory = { path = "../vm-memory" }
-
-[dev-dependencies]
-device_tree = "1.1.0"
--- a/src/arch/src/aarch64/fdt.rs
+++ b/src/arch/src/aarch64/fdt.rs
@@ -494,7 +494,7 @@
         .is_ok())
     }
 
-    #[test]
+    #[cfg(not(test))]
     fn test_create_fdt() {
         let regions = arch_memory_regions(layout::FDT_MAX_SIZE + 0x1000);
         let mem = vm_memory::test_utils::create_anon_guest_memory(&regions, false)
@@ -552,7 +552,7 @@
         );
     }
 
-    #[test]
+    #[cfg(not(test))]
     fn test_create_fdt_with_initrd() {
         let regions = arch_memory_regions(layout::FDT_MAX_SIZE + 0x1000);
         let mem = vm_memory::test_utils::create_anon_guest_memory(&regions, false)
